packages/fluentui/react-northstar/src/themes/teams/staticStyles/globalStyles.ts

```diff
@@ -17,4 +17,8 @@ export const globalStyles: StaticStyleFunction = siteVars => ({
   '*:after': {
     boxSizing: 'border-box',
   },
+  /* Adding priority for HTML `hidden` attribute to be applied correctly */
+  '[hidden]': {
+    display: 'none!important',
```

Comment:
A bit scary to use !important in global styles, but I don't have a better idea for this..

Reply:
Yeah, from what I've found there is nothing much else to do for that case :( 
Some of other CSS frameworks are doing the same